web_monitoring/diffing_server.py

```diff
@@ -3,14 +3,26 @@
 import hashlib
 from importlib import import_module
 import inspect
-import json
 import tornado.gen
 import tornado.httpclient
 import tornado.ioloop
 import tornado.web
 import web_monitoring
 
 
+# MAP tokens in the REST API to functions in modules.
+# The modules do not have to be part of the web_monitoring package.
+CONFIG = {
```

Comment:
I feel like a different name might be clearer at this point, e.g. `DIFF_ROUTES` or `DIFF_ENDPOINTS` or something, this this is really just a list of URL paths and doesn’t hold any other sorts of things one might think of as configuration.